HW9/src/test/java/UserTest.java

```diff
@@ -0,0 +1,126 @@
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+import java.util.ArrayList;
+import java.util.List;
+
+public class UserTest {
+    private User user1;
+    private User user2;
+    private User user3;
+    private User userNotAdd;
+    private User userNotAdd1;
+    private User userStringAdd;
+
+    @Before
+    public void setUp () {
+        user1 = new User("Stepan", 21, Sex.MALE);
+        user2 = new User("Lera", 19, Sex.FEMALE);
+        user3 = new User("Alina", 12, Sex.FEMALE);
+        userNotAdd = new User("", 0, null);
+        userNotAdd1 = new User(null, 0, null);
+    }
+
+    @Test
+    public void getAllUsers () {
+        List <User> expected = User.getAllUsers();
+        List <User> actual = new ArrayList <>();
+        actual.add(user1);
+        actual.add(user2);
+        actual.add(user3);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void getAllUsersNoNULL () {
```

Comment:
It is not camelStyle? should be  getAllUsersNoNULL->  getAllUsersNotNull
The same about other methods. 
But better getAllUsersNoNULL->  shouldReturnAllUsersNotNull